src/lib/components/backoffice/Sidebar/Sidebar.js

```diff
@@ -59,155 +67,125 @@ class Sidebar extends Component {
                 <Menu.Item>
                   <Menu.Header>Actions</Menu.Header>
                   <Menu.Menu>
-                    <Menu.Item
-                      as={Link}
-                      active={checkInActive}
-                      to={BackOfficeRoutes.checkIn}
-                    >
-                      Check-in
-                      <div className="menu-item-description">Return copies</div>
-                    </Menu.Item>
-                    <Menu.Item
-                      as={Link}
-                      active={checkOutActive}
-                      to={BackOfficeRoutes.checkOut}
-                    >
-                      Check-out
-                      <div className="menu-item-description">Create loan</div>
-                    </Menu.Item>
+                    {MenuItem(
```

Comment:
sorry, I will be a bad guy here a bit. 
It is more "react way" to have a component instead of using the JS function, and the function does exactly what `<Menu.Item>` component does, but in terms of the code it is less readable IMO, f.e. because the function arguments are not named as the component props.